### PR TITLE
Adding Keras CNN model

### PR DESCRIPTION
**Pull request by nthain**

### CODE REVIEW

File: experiments/tf_trainer/keras_cnn/model.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        name='comment_text')
+
+    # Convolutional Layers
+    X = I
+    for filter_size in self.hparams().filter_sizes:
+        X = layers.Conv1D(self.hparams().num_filters, filter_size, activation='relu', padding='same')(X)
+
+
+    X = layers.GlobalAveragePooling1D()(X)
+    #X = layers.Flatten()(X)
```
Comment (jjtan) [inline experiments/tf_trainer/keras_cnn/model.py]: remove?

File: experiments/tf_trainer/keras_cnn/model.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    # Outputs
+    outputs = []
+    for label in self._labels:
+      outputs.append(layers.Dense(1, activation='sigmoid', name=label)(X))
+
+    model = models.Model(inputs=I, outputs=outputs)
+    model.compile(
+        optimizer=optimizers.Adam(lr=self.hparams().learning_rate),
+        loss='binary_crossentropy',
+        metrics=['accuracy', super().roc_auc])
```
Comment (jjtan) [inline experiments/tf_trainer/keras_cnn/model.py]: this might be where my bug is... I don't think you should wait, but you may have to change this later to support multihead

File: experiments/tf_trainer/keras_cnn/run_ml_engine.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```
Comment (jjtan) [inline experiments/tf_trainer/keras_cnn/run_ml_engine.sh]: Random note: seeing such a similar file is motivating me to figure out some way to make this common

File: experiments/tf_trainer/keras_cnn/model.py (hunk 1 above)
Comment (nthain) [reply experiments/tf_trainer/keras_cnn/model.py]: Done.

**Comment by jjtan** [comment]
Maybe you aren't running into the issue then.. What I was expecting was really bad roc_aucs. Accuracy might be a separate problem / might be a metric problem (you can try 'binary_accuracy' maybe)
